### DISCUSSION TITLE
"Physics at the limits of reality" Sabine Hossenfelder

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#212] [Metaphysics and Philosophy]
"Physics at the limits of reality" Sabine Hossenfelder in conversation with Hilary Lawson
[https://youtu.be/6xGwdUCYzgw](https://youtu.be/6xGwdUCYzgw)

**Comment by OlliSant** [comment]
Cosmology is the physics and philosophy of the universe outside of the visible universe.

**Comment by RedshiftDrift** [comment]
***Future generations will find our science ridiculous | Philosopher Hilary Lawson***

Lawson doesn't know, this generation already finds our "science" ridiculous!
45 seconds https://youtu.be/S9ZYM_RNblE
"Our account of the world - Big Bang, inflation, dark energy, dark matter... that was obviously totally ridiculous."
